contributors/guide/README.md

```diff
@@ -208,7 +208,7 @@ If you haven't noticed by now, we have a large, lively, and friendly open-source
 
 ## Events
 
-Kubernetes is the main focus of CloudNativeCon/KubeCon, held twice per year in EMEA and in North America. Information about these and other community events is available on the CNCF [events](https://www.cncf.io/events/) pages.
+Kubernetes is the main focus of KubeCon + CloudNativeCon, held thrice per year in China, EMEA and in North America. Information about these and other community events is available on the CNCF [events](https://www.cncf.io/events/) pages.
```

Comment:
I'd suggest "held three times per year" over "held thrice per year".

Reply:
Should EMEA also be replaced by Europe as the conferences are called KC+CNC China / Europe / North America?

Reply:
Changed and squashed it.